src/migrator/migrations.sbvr

```diff
@@ -21,3 +21,52 @@ Term: migration lock
 
 Fact Type: migration lock has model name
 	Necessity: each migration lock has exactly one model name
+
+Term: migration key
+	Concept Type: Short Text (Type)
+Term: start time
+	Concept Type: Date Time (Type)
+Term: last run time
+	Concept Type: Date Time (Type)
+Term: run counter
+	Concept Type: Integer (Type)
+Term: migrated rows
+	Concept Type: Integer (Type)
+Term: error counter
+	Concept Type: Integer (Type)
+Term: converged time 
+	Concept Type: Date Time (Type)
+Term: last error message
+	Concept Type: Text (Type)
+
+Term: migration status
+	Reference Scheme: migration key
+	Database ID Field: migration key
+
+Fact Type: migration status has migration key
+	Necessity: each migration status has exactly one migration key
+
+Fact Type: migration status has start time
+	Necessity: each migration status has at most one start time
+
+Fact Type: migration status has last run time
+	Necessity: each migration status has at most one last run time
+
+Fact Type: migration status has run counter
+	Necessity: each migration status has at most one run counter
+
+Fact Type: migration status has migrated rows
+	Necessity: each migration status has at most one migrated rows
+
+Fact Type: migration status has error counter
+	Necessity: each migration status has at most one error counter
+
+Fact Type: migration status is backoff
+
+Fact Type: migration status has converged time 
+	Necessity: each migration status has at most one converged time
+
+Fact Type: migration status has last error message
+	Necessity: each migration status has at most one last error message
+
+Fact Type: migration status should stop
```

Comment:
@thgreasi this would be some kind of debugging tool to understand why the async migrations fail.
I'll double check if the same message will be in the logs as well and will then delete the column/field
Thanks!

Reply:
@thgreasi it will be somewhere in the service logs. Skipping it in the DB.
Thanks for catching it!

Reply:
@fisehara I meant that I don't see the `migration status should stop` being used anywhere.

Reply:
@thgreasi it was meant to be a DB queriable information during debugging on production database. But as this information is also logged to the api system logs it's redundant and is not needed to be persisted in the DB.

Reply:
I think the point is that you can remove it from here :P

Reply:
Oh right... I was mixing it up with the last error message, which I deleted already. Yes the 'should stop' can be deleted.